docs/csharp/how-to/search-strings.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "How to: search strings (C# Guide)"
+ms.date: 02/21/2018
+ms.prod: .net
+ms.technology: 
+  - "devlang-csharp"
+ms.topic: "article"
+helpviewer_keywords: 
+  - "searching strings [C#]"
+  - "strings [C#], searching with String methods"
+ms.assetid: fb1d9a6d-598d-4a35-bd5f-b86012edcb2b
+ms.author: "wiwagn"
+---
+
+# How to: search strings
+
+You have two main strategies that you can use to search for text in strings. Member methods of the <xref:System.String> class search for specific text. Regular expressions search for patterns in text.
+
```

Comment:
To be consistent with other two "How to" on strings, we might add a note on interactive C# code blocks:
`[!INCLUDE[interactive-note](~/includes/csharp-interactive-note.md)]`